elections/steering/2023/candidate-pacoxu.md

```diff
@@ -0,0 +1,47 @@
+In SIG Node, my contributions span various aspects of node management. I have fixed issues related to node resources, and CI failures, and have also upgraded and improved functionalities to enhance node stability and performance.
+
+I've been one of the [most active contributors](https://k8s.devstats.cncf.io/d/13/developer-activity-counts-by-repository-group?orgId=1&var-period_name=Last%203%20years&var-metric=contributions&var-repogroup_name=Kubernetes&var-repo_name=kubernetes%2Fkubernetes&var-country_name=All) to the project over the years.
+
+I won the [2021 Kubernetes Contributor Award](https://www.kubernetes.dev/community/awards/2021/) for both SIG-ClusterLifecyle and SIG-CLI. And I organized KCD Chengdu 2022, and am a speaker in Kubecon EU 2023 and Kubecon China 2021. I am a member of the Program Committee for KubeCon China 2023. My speech in KCD Shanghai 2022 is about an introduction about SIG-Node & kubeadm, and how to participate.
+
+As I worked in China, we released that there are many new contributors in China that cannot get a timely response in the community for the timezone reason, and we met the problem at the beginning as well. We helped them know the process and guide them to complete their contributions. I also helped in the Kubernetes Contributor Summit in China this year.
+
+For WG-LTS, we started a [KLTS project](https://github.com/klts-io/kubernetes-lts) in 2021 and will participate in revived WG LTS.
+
+## What I'll do
+
+Serving in the Steering Committee, I think we will be an important interface between the project and CNCF. I will also maintain the governance documents and help to build paths within the community, among SIGs or WGs.
+
+For daily contributions, I'm the leader of the open-source team in DaoCloud. As time goes on, I and my team will continue to do more within the community, driving the development and progress of the Kubernetes ecosystem. I will be in In-Person Kiosk at Kubecon China for SIG-Node Kubernetes Community. We will encourage attendees to contribute to the projects and answer questions about the community. Furthermore, I will continue to support new contributors within the community, especially those from China and working in APAC, helping them familiarize themselves with the processes and encouraging their active involvement. The project's health and new contributors are quite important to the community.
+
+I deeply understand the responsibilities and obligations that come with being a PMC member, and I will approach this role with a positive attitude, a spirit of collaboration, and a continuous commitment to contribute to the project's long-term development and success. I have a passion for the Kubernetes community and hope to be given the opportunity to serve as a PMC member, contributing my efforts toward the prosperity and growth of the project.
```

Comment:
In this section you mention being a PMC member a couple of times - do you want to change that to being a Steering member? Or is there something else that would help explain?

Reply:
Sorry for the mistake. Updated.